kafka/docker-compose.yml

```diff
@@ -4,8 +4,12 @@ services:
     image: core/zookeeper
     ports:
       - "2181:2181"
+    environment:
+      - HAB_LICENSE=accept-no-persist
```

Comment:
This might need to refer to another variable that we require the user to set - it might be bad to hardcode license acceptance into config.
@smacfarlane what do you reckon?

Reply:
Added.